packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -105,27 +130,36 @@ export class Checkbox extends React.Component<ICheckbox> {
       id,
       selected,
       label,
+      hint,
       value,
       onChange,
       size = 'small'
     } = this.props
     return (
-      <Wrapper>
-        <Input
-          id={id}
-          role="checkbox"
-          checked={selected}
-          type="checkbox"
-          name={name}
-          value={value}
-          onChange={onChange}
-          size={size === 'large' ? 40 : 16}
-        />
-        <Check size={size}>
-          {selected && (size === 'large' ? <TickLarge /> : <Tick />)}
-        </Check>
-        <Label htmlFor={id}>{label}</Label>
-      </Wrapper>
+      <>
+        {hint && (
+          <Text variant={'reg16'} element={'p'} color={'grey500'}>
+            {hint}
+          </Text>
+        )}
```

Comment:
I think as @jpye-finch suggested [here](https://opencrvsworkspace.slack.com/archives/C02UN28EJ9M/p1669812181905929?thread_ts=1669808560.574499&cid=C02UN28EJ9M) the "hint" should apply to CheckboxGroup -component rather than Checkbox. We should also add a new story with "Checkbox group with hint".

Also we should add a "label" -prop for the CheckboxGroup, which is the black text over the hint.

Reply:
Okay